<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>31</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>45</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="search_filter_description">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Find</string>
     </property>
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="label_filter">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="case_sensitive">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Case sensitive</string>
     </property>
     <property name="text">
      <string>Aa</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="regexp">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Regular expression</string>
     </property>
     <property name="text">
      <string>.*</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="result">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>No results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="arrow_left">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Previous match</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="arrow_right">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Next match</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
